Sources/TuistKit/Services/TestService.swift

```diff
@@ -109,11 +110,17 @@ final class TestService {
 
         let testableSchemes = buildGraphInspector.testableSchemes(graphTraverser: graphTraverser) +
             buildGraphInspector.projectSchemes(graphTraverser: graphTraverser)
+        let testableSchemesString = Set(testableSchemes.map(\.name)).joined(separator: ", ")
         logger.log(
```

Comment:
nitpick: I'd move this log into `else` branch [here](https://github.com/tuist/tuist/pull/3799/files#diff-f6388c2ef94259dd1757eaee2f70badee66cba09012cfb07a9ea87501721b983R119). This will be printed twice if you ran this in `debug`.

Reply:
Agree, good shout